internal/packager/registry.go

```diff
@@ -86,6 +98,38 @@ func Push(app *types.App, namespace, tag, repo string) (string, error) {
 	if namespace != "" && namespace[len(namespace)-1] != '/' {
 		namespace += "/"
 	}
-	imageName := namespace + repo + ":" + tag
-	return resto.PushConfigMulti(context.Background(), payload, imageName, resto.RegistryOptions{}, nil)
+	return namespace + repo + ":" + tag
+}
+
+func createPayload(app *types.App) (map[string]string, error) {
+	payload := map[string]string{
+		internal.MetadataFileName: string(app.MetadataRaw()),
+		internal.ComposeFileName:  string(app.Composes()[0]),
+		internal.SettingsFileName: string(app.SettingsRaw()[0]),
+	}
+	err := readAttachments(payload, app.Path, app.Attachments())
+
+	return payload, err
```

Comment:
In Go, it's assumed that you should ignore results if `err != nil` but it's relatively common practice to return a `nil` (or empty) result in the case of an error. In this case that would mean:
```go
if err := readAttachments(payload, app.Path, app.Attachments()); err != nil {
    return nil, err
}
return payload, nil
```